admin/lib/generators/solidus_admin/install/install_generator.rb

```diff
@@ -24,6 +26,21 @@ def ignore_tailwind_build_files
       def build_tailwind
         rake "solidus_admin:tailwindcss:build"
       end
+
+      def install_lookbook
+        return unless options[:lookbook]
+
+        # The dummy app has no Gemfile
+        return unless in_root { File.exist?("Gemfile") }
```

Comment:
Nit, but maybe we can now remove these lines, as we won't provide `SOLIDUS_ADMIN_LOOKBOOK` variable for the dummy app.